pubspec.yaml

```diff
@@ -11,8 +11,8 @@ environment:
   sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
+  pedantic: ^1.9.0
```

Comment:
See the comment above - I think we should keep pedantic as a dev dependency.

Reply:
Done